app/views/companies/_search_form.html.haml

```diff
@@ -0,0 +1,49 @@
+= search_form_for @search_params, method: :get, url: companies_path,
+                                        class: 'form-horizontal' do |f|
+  .form-group
+    .col-sm-5
+      = f.label :business_categories_id_in,
+                "#{t('activerecord.models.business_category.one')}",
+                class: 'control-label',
+                style: 'text-align: left; font-size: 12px;'
+      = f.collection_select :business_categories_id_in,
```

Comment:
Sort of an aside -- does this point to a need for some better testing data?  
I'm running into that.  For me, it'd be helpful to have a richer set of data to play with.  Ex:  more membership applications, in different states, with different categories, etc.